dask_image/ndinterp/__init__.py

```diff
@@ -105,21 +107,21 @@ def affine_transform(
         offset = matrix[:image.ndim, image.ndim]
         matrix = matrix[:image.ndim, :image.ndim]
 
+    cval = kwargs.pop('cval', 0)
+    mode = kwargs.pop('mode', 'constant')
+    prefilter = kwargs.pop('prefilter', False)
+    if mode in ['wrap', 'reflect', 'mirror', 'grid-mirror', 'grid-wrap']:
+        raise NotImplementedError(f"Mode {mode} is not currently supported.")
```

Comment:
Any chance you can add what modes we *do* support to that error message? That's likely to be a lot less frustrating to users when they run into it. (And yes, I realize you didn't write this message originally, but if it's an easy fix I'd like us to do it)

Reply:
Looks like 'constant' and 'nearest' are the two safe modes

Reply:
yes. also 'grid-constant' if using SciPy 1.6+.